src/ripple/app/misc/ValidatorList.h

```diff
@@ -406,7 +412,7 @@ class ValidatorList
         std::string const& manifest,
         std::uint32_t version,
         std::vector<ValidatorBlobInfo> const& blobs,
-        std::string siteUri,
+        std::string source,
```

Comment:
There are only two non-test callers to `applyListsAndBroadcast`, in ValidatorSite.cpp (line 405), and PeerImp.cpp (line 2298). Instead of the overhead of parsing `source` to figure out what type it is, why not add another parameter to this function, and `applyLists` and `applyList`? To keep it super explicit, you could make it an `enum`:
```
enum class SourceType
{
  Site,
  Peer
};
```